### ISSUE TITLE
Where should I do the PR for upgrading mvc demo

### ISSUE THREAD

**Issue by Jogai** [NightOwl888/BoboBrowse.Net#33]
If you're even interested, I upgraded the demo to mvc5. Which branch do you want to do this pull request on?

**Comment by NightOwl888** [comment]
Yes, I would be interested in seeing it, especially if you have done the port on 4.0.2 and it works. Since cardemo was deleted from bobo-browse 4.x, it was just guesswork trying to port the demo. I managed to make the pieces fit several years ago, but there was some issue that I don't quite recall - it had something to do with the selection boxes not populating if I remember correctly. Since the 4.0.2 branch has already been merged, please target the master branch.

**Comment by Jogai** [comment]
I didnt port to 4x. Just the project to mvc5 and working for vs2019.
